homeassistant/components/zha/websocket_api.py

```diff
@@ -582,8 +583,9 @@ async def websocket_add_group_members(
         return
 
     await zha_group.async_add_members(members)
-    ret_group = zha_group.group_info
-    connection.send_result(msg[ID], ret_group)
+    ret_group = zha_gateway_proxy.get_group_proxy(group_id)
+    assert ret_group
```

Comment:
I think some reviews I've seen in the past mentioned to not use asserts in HA production code at all, but it's already used in the other methods and other integrations do too, so probably fine. Maybe I just misremembered..